core/scripts/cli/cli_entry.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
```

Comment:
`set -e`? E.g., in case the compile failed, we probably want to stop immediately.

Reply:
Agreed.